### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by apacios**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,21 @@
 
 ## About
 
-With this module we can export all the translatable sentences and words in a CSV file.
-Also, we can import a CSV file to import all the translations we want.
+With this module we can export all the translatable sentences and words in a XLSX file.
+Also, we can import a XLSX file to import all the translations we want.
 
 ## Actions
```
Comment (Quetzacoalt91) [inline README.md]:
```suggestion
## Features 
```

File: README.md [hunk 2]
```diff
@@ -2,15 +2,21 @@
 
 ## About
 
-With this module we can export all the translatable sentences and words in a CSV file.
-Also, we can import a CSV file to import all the translations we want.
+With this module we can export all the translatable sentences and words in a XLSX file.
+Also, we can import a XLSX file to import all the translations we want.
 
 ## Actions 
 * Export the translatable translations
```
Comment (Quetzacoalt91) [inline README.md]:
```suggestion
 * Export the translatable strings
```

or 

```suggestion
 * Export the translations
```

File: README.md (hunk 1 above)
Comment (apacios) [reply README.md]: Thanks

File: README.md (hunk 2 above)
Comment (apacios) [reply README.md]: Thanks

**Comment by Quetzacoalt91** [comment]
Thanks @apacios

**Pull request merged**